cmd/ak/cmd/configuration/where.go

```diff
@@ -19,11 +21,23 @@ var whereCmd = common.StandardCommand(&cobra.Command{
 
 	RunE: func(cmd *cobra.Command, args []string) error {
 		cfg := xdg.ConfigHomeDir()
+
+		if onlyCfg {
+			fmt.Fprintln(cmd.OutOrStdout(), cfg)
+			return nil
+		}
+
```

Comment:
Move your block under the `if strings.Contains(cfg, " ")` - so if `cfg` contains spaces you'll print `"path"` instead of `path`. Same with `data` below.